Sources/Core/ObjCModelRenderer.swift

```diff
@@ -120,6 +120,20 @@ public struct ObjCModelRenderer: ObjCFileRenderer {
     }
 
     func renderRoots() -> [ObjCIR.Root] {
+        let enumProperties = properties.filter { (arg) -> Bool in
+            let (_, schema) = arg
+            switch schema.schema {
+            case .enumT:
+                return true
+            default:
+                return false
+            }
+        }
+        let enumIVarDeclarations: [(Parameter, TypeName)] = !enumProperties.isEmpty ? enumProperties.compactMap { (arg) -> (Parameter, TypeName) in
```

Comment:
Do we need the isEmpty check here? I assume enumProperties will be an array regardless and if it's empty we'll return the `[]` value anyways.

Reply:
No. I'll update.